NMS-14212: Event docs cleanup (#5002)
OpenNMS/opennms: docs/modules/operation/pages/events/introduction.adoc

[[events]]
= Events

Events are structured historical records of things that happen in {page-component-title} and the nodes, interfaces, and services it monitors.
Events are central to the operation of the {page-component-title} platform, so it's critical to have a firm grasp of this topic.

> Whenever something in {page-component-title} appears to work by magic, it's probably events working behind the curtain.

Reviewer: ```suggestion
Whenever something in {page-component-title} appears to work by magic, it's probably events working in the background.
```

Events may originate within {page-component-title} itself, or from an external source via <<events/sources/snmp-traps.adoc#ga-events-sources-snmp-traps, SNMP traps>>, <<events/sources/syslog.adoc#ga-events-sources-syslog, syslog messages>>, or a variety of other <<events/event-sources.adoc#ga-events-sources-of-events, sources>>.
Event definition files are used to provide a standard framework for Eventd to process events evenly, regardless of how they were sent to the server.

[[ga-events-event-bus]]
== The event bus

At the heart of {page-component-title} lies an event bus.
Any {page-component-title} component can publish events to the bus, and any component can subscribe to receive events of interest that have been published on the bus.
This publish-subscribe model enables components to use events as a mechanism to send messages to each other.

> For example, the provisioning subsystem of {page-component-title} publishes a node-added event whenever a new node is added to the system.

Reviewer: ```suggestion
For example, the {page-component-title} provisioning subsystem publishes a node-added event whenever a new node is added to the system.
```

[[events]]
= Events

Events are central to the operation of the {page-component-title} platform, so it's critical to have a firm grasp of this topic.

> IMPORTANT: Whenever something in {page-component-title} appears to work by magic, it's probably events working behind the curtain.

Reviewer 2: We can remove the important, and just have this as a second sentence.

Author: ```suggestion
Whenever something in {page-component-title} appears to work by magic, it's probably events working behind the curtain.
```
